include/freerdp/client/rdpsnd.h

```diff
@@ -37,9 +37,7 @@ struct _RDPSND_WAVE
 	UINT16 wTimeStampA;
 	UINT16 wTimeStampB;
 
-	UINT16 wLatency;
```

Comment:
that variable was practical, and as a matter of fact, it would break it for the wayk client. I'd rather get it back than remove it.